cli/command_server_start.go

```diff
@@ -273,6 +273,19 @@ func (c *commandServerStart) run(ctx context.Context) error {
 	return errors.Wrap(srv.SetRepository(ctx, nil), "error setting active repository")
 }
 
+// shutdownServer shutdown http server and close the repository.
+//
+//nolint:revive,unparam
```

Comment:
Note, disabling revive completely disables _all_ the rules of revive. We should only disable particular rules, if we need. See the docs: https://github.com/mgechev/revive#comment-directives